src/lint/collect-nodes.ts

```diff
@@ -140,3 +140,15 @@ export function collectNodes(
 
   return { success: true, mainGrammar, headers, sdos, earlyErrors, algorithms };
 }
+
+function textContentExcludingDeleted(node: Node): string {
+  let retval = '';
+  node.childNodes.forEach(value => {
+    if (value.nodeType === 3) {
+      retval += value.nodeValue;
+    } else if (value.nodeType !== 1 || (value as Element).tagName !== 'DEL') {
```

Comment:
I think you should only descend into [flow content](https://html.spec.whatwg.org/multipage/dom.html#flow-content) (excluding `<template>`?). Not sure the easiest way to do that.

Reply:
Ehh, I don't think that's worth worrying about in this PR.